src/noaa_apt.rs

```diff
@@ -150,17 +149,20 @@ pub fn process(
 
     use image::GrayImage;
 
-    let mut img: GrayImage = GrayImage::from_vec(
-            PX_PER_ROW, height, map(&signal, low, high))
-        .ok_or_else(|| err::Error::Internal(
-            "Could not create image, wrong buffer length".to_string()))?;
-    
+    let mut img: GrayImage = GrayImage::from_vec(PX_PER_ROW, height, map(&signal, low, high))
+        .ok_or_else(|| {
+            err::Error::Internal("Could not create image, wrong buffer length".to_string())
+        })?;
+
     if let Contrast::Histogram = contrast_adjustment {
         img = processing::histogram_equalization(&img)?;
     }
 
     let mut img: Image = image::DynamicImage::ImageLuma8(img).into_rgba(); // convert to RGBA
 
+    if false_color {
```

Comment:
I mean, right now, when histogram equalization is selected:

- First a "MinMax adjustment"  is made
- Histogram equalization is made
- False color is done

It would be better to do:

- First a "90 percent" (I don't like "From telemetry" because it won't work in short images)
- Do False color
- Do histogram equalization

But I don't know if histogram equalization can be done easily in color images. So it is enough if you leave it as it is now